src/pages/DataSharingLanguageTool.js

```diff
@@ -54,7 +54,7 @@ export default function DataSharingLanguageTool() {
     }
     await DataUseTranslation.translateDataUseRestrictions(dataUse)
       .then((resp) => { translatedDataUse = resp; });
-    translatedDataUse = translatedDataUse.forEach((sentence) => {
+    translatedDataUse.forEach((sentence) => {
```

Comment:
This is technically a functional change, but appears to have no impact on actual functionality.